src/caffe/layers/image_data_layer.cpp

```diff
@@ -46,6 +46,9 @@ void ImageDataLayer<Dtype>::DataLayerSetUp(const vector<Blob<Dtype>*>& bottom,
     lines_.push_back(std::make_pair(line.substr(0, pos), label));
   }
 
+  // Check that image list is not empty.
+  CHECK_GT(lines_.size(), 0);
+
```

Comment:
Just edit this PR. You need to `git commit --amend` your patch and then push force to your branch and it will automatically update this PR.